### ISSUE TITLE
From Joomla 4.4.1 - Notes on the update source ...could not be found

### ISSUE THREAD

**Issue by WM-Loose** [joomla/joomla-cms#42421]
Who came up with this again and didn't think about the consequences?

We receive inquiries here in the Joomlaplates forum as to why these errors for our template updates and modules are suddenly displayed in the backend of J4.4.1.

Previously, our existing updates were always displayed, which were not provided on the update server and only had to be downloaded with a valid subscription and then installed in the backend. How should I set this in the backend so that it is displayed as before. I didn't find a function for activation or deactivation.

I can't deactivate all the update sources now and no longer receive any notifications?

No matter what, PLEASE turn it back quickly.

Execution:
Backend /Extensions /Check for updates

Result:
![1](https://github.com/joomla/joomla-cms/assets/99647423/ff683764-b586-4be6-a93a-68c18ce059a0)

**Comment by brianteeman** [comment]
> Who came up with this again and didn't think about the consequences?

I think what you meant to say is that we are not following the documented process for extension updates and what can we do to fix that.

**Comment by WM-Loose** [comment]
Wow, it only seems to affect Joomla installations at Strato and 1und1. 
Can you possibly confirm that?

**Comment by WM-Loose** [comment]
The culprit and problem is clearly the hoster Strato. We installed the Joomla installation 1:1 with another provider (DB version and PHP version identical) and there were no more errors when retrieving the update server. The website also has significantly faster loading times than Strato. We found out that the provider has probably been struggling with network problems for a long time. We will recommend the customer to choose another provider.
